src/microbe_stage/MicrobeHUD.cs

```diff
@@ -267,6 +267,7 @@ protected override void ShowFossilisationButtons()
 
             var button = FossilisationButtonScene.Instantiate<FossilisationButton>();
             button.AttachedEntity = entity;
+            button.IsMicrobeStage = true;
```

Comment:
Just me wondering: I guess as multicellular doesn't seem to have its own HUD so no change related to that is required.